build_fuzz_targets.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
```

Comment:
Hi, thanks for the PR! After looking at this, I think it might be better to have this code inside of the `CMakeLists.txt` instead of as a separate build script. It's working fine, but I want to make sure that the upstream also accepts this changes, so we should make sure that it's consistent with the build process of the upstream. Could you please
1. Move the fuzz target behavior into the CMakeLists (perhaps with a target name of `mayhem-fuzz`):
2. Remove this script.
3. Update the Dockerfile to use CMake instead of the build script.
That should be it! Thanks again for your work!

Reply:
I have pushed a new commit. Please check it and give any suggestions.

Reply:
LGTM, thanks!